wicket-core/src/main/java/org/apache/wicket/markup/html/form/AutoLabelResolver.java

```diff
@@ -134,15 +139,15 @@ private String getWicketNamespace(MarkupStream markupStream)
 
 	/**
 	 * 
-	 * @param container
-	 * @param id
+	 * @param container The container
+	 * @param path The path of the component

```

Comment:
The `relative` (?!) path to the component ?

Reply:
yes... better as you describe it.

Reply:
Fixed